### ISSUE TITLE
Atom update changed styling?

### ISSUE THREAD

**Issue by alizahid** [atom-material/atom-material-ui#413]
I'm not sure which version, but it was a few weeks ago. The issue is with padding and font size of some elements.

![](http://i.imgur.com/2o3MQdH.png)

In the screenshot, you can see the very compact tree view. I've scoured through editor, core, theme, (even) syntax settings, but I couldn't find a way to make them fat again.

Also, this could be paranoia, but the font of the tab titles looks weird? Is it a different font from the rest of the theme, or just a lighter weight?

![](http://i.imgur.com/Qt6ZEWm.png)

Next, you'll see the Settings pane with the correct padding for items. But the packages now lack padding?

Maybe an Atom update changed styling and now the theme looks different? Or am I somehow dreaming all this?

![](http://i.imgur.com/I4q8CL1.jpg)

**Comment by silvestreh** [comment]
If you updated to Atom 1.17 recently you probably also upgraded to Material UI 2.0 which is a complete rewrite of the theme. Things will look different, so it's not just you.

**Comment by alizahid** [comment]
Yes to both. Well, that's a relief. Any chance we'll see an option for the treeview padding soon? Or if you can tell me the class name, I can add it to my stylesheet.

**Comment by PeteCrighton** [comment]
+1 for an option for the non-compact tree view.

**Comment by silvestreh** [comment]
Yeah, I'm working on a patch to restore the fat tree-view items and an option to make them slim.

**Comment by alexindigo** [comment]
It broke my tabs bar, it's either only icons for the horizontal view, for only-one-visible line + scroll for the vertical view.

**Comment by anthony-pinskey** [comment]
@silvestreh Do you know if the update also changed how Typescript looks? The syntax highlighting looks so much different in Atom 1.19+ now. I kept thinking something was broken.
